src/client/app/components/MultiSelectComponent.jsx

```diff
@@ -53,9 +53,16 @@ MultiSelectComponent.propTypes = {
 		label: PropTypes.string.isRequired,
 		value: PropTypes.number,
 	})),
-	selectedOptions: PropTypes.arrayOf(PropTypes.shape({
-		label: PropTypes.string.isRequired,
-		value: PropTypes.number,
-	})),
+	selectedOptions: PropTypes.oneOfType([
```

Comment:
Selected options is either an array of options or a single option? That seems unintuitive. It should probably be an array that can possibly just have a single item in it.

Reply:
This is how the Select component works in react-bootstrap

Reply:
Actually two different things using multiSelect component are passing objects and array. So, to accomodate both, we had to do this. Otherwise, it was throwing errors.

Reply:
Fair enough.